src/Uno.UI/UI/Xaml/FrameworkElement.wasm.cs

```diff
@@ -43,7 +43,7 @@ internal sealed override void ManagedOnLoading()
 			}
 			catch (Exception error)
 			{
-				this.Log().Error("ManagedOnLoading failed in FrameworkElement", error);
+				_log.Error("ManagedOnLoading failed in FrameworkElement", error);
```

Comment:
This logging technique will not allow for dynamic turn on of logging. May be problematic in the future.